recipes/libsystemd/all/conanfile.py

```diff
@@ -128,6 +132,11 @@ def build(self):
                                    project_source_root).stdout().strip()""",
             "relative_source_path = '../{}'".format(self._source_subfolder))
 
+        if tools.Version(self.version) >= "248":
+            missing_syscals = os.path.join(self._source_subfolder, "src",
```

Comment:
What is this fixing? An unicode decoding error for python 2?
Can this be moved to a patch?

Reply:
When building the recipe inside `conanio/gcc49` container, meson configuration is failing on `find_program('missing_syscalls.py')` with error:
```
'ascii' codec can't decode byte 0xe2 in position 615: ordinal not in range(128)
Unusable script '/home/conan/.conan/data/libsystemd/248.3/_/_/build/3054d372299ea1c8d60348e3f59b59092e11091c/source_subfolder/src/basic/missing_syscalls.py'
Program missing_syscalls.py found: NO
```
It looks like problem with python 2 and unicode but nothing should be executed with python 2 in this case. I'm not sure what exactly causes this error.

Reply:
Thanks for the answer!